### DISCUSSION TITLE
Instance sometimes not reverting status to idle after running

### DISCUSSION THREAD

**Discussion by jramke** [pockethost/pockethost#281] [Q&A]
Hey,

sometimes after the instance started correctly when i loaded my live site it is not getting back to idle although actually no more requests are made. 

This results in a usage of 1443 mins now wich i want to avoid at latest when pockethost is going to v1.

I dont really know if its a pockethost bug or if i fucked something up in my code. Im using Sveltekit for the first time, maybe i missed some basics. 
I already tried to use cache to prevent unnecessary pocketbase requests. Im using api routes and doing the requests in the exported GET function and fetching it in my layouts load function where i set cache-control to 3600 using the setHeaders method.

Do I understand the screenshot correctly that the instance was active for almost 24 hours?
![Bildschirmfoto 2023-09-07 um 08 45 41](https://github.com/benallfree/pockethost/assets/81250358/582122e0-913f-489f-a035-900eb02a6e83)

My site is hosted on vercel.

Maybe someone got an idea what it could be.
Thanks for every reply.

**Comment by benallfree** [comment]
Thanks for the report. We'd need to dig more to isolate whether this was a PocketHost issue or something else. Are you using any realtime features of PocketBase?

**Comment by jramke** [comment]
No, I’m just loading collections and file urls (it's a small portfolio site)

**Reply by benallfree** [reply]
Other than accumulating usage, is it affecting anything?

**Reply by jramke** [reply]
No I don't think so, maybe just some slightly longer loading times because I'm over the 100mins I guess.

**Reply by benallfree** [reply]
PocketHost doesn’t do any rate limiting.

**Reply by jramke** [reply]
Oh ok, then nothing else is affected. I thought I read something in the faq where the instance is moved to to a queue or something after the 100 mins of free usage but maybe that's an outdated info.
